cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,66 @@
-describe('', () => {
-  before(() => {
+/// <reference types='cypress' />
 
-  });
+describe('Article flow', () => {
+  let user;
+  let article;
 
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('/');
+    cy.task('generateUser')
+      .then((generateUser) => {
+        user = generateUser;
+      });
 
+    cy.task('generateNewArticle')
+      .then((generateNewArticle) => {
+        article = generateNewArticle;
+      });
   });
 
-  it('', () => {
+  it('should create the article', () => {
+    cy.login(user.email, user.username, user.password);
+    cy.visit('/editor');
+    cy.findByPlaceholder('Article Title')
+      .type(article.title);
+    cy.findByPlaceholder('What\'s this article about?')
+      .type(article.description);
+    cy.findByPlaceholder('Write your article (in markdown)')
+      .type(article.body);
+    cy.findByPlaceholder('Enter tags')
+      .type(article.tag);
+    cy.contains('button', 'Publish Article')
+      .click({ force: true });
+    cy.contains('h1', article.title)
+      .should('be.visible');
+    cy.url()
+      .should('include', 'article');
+  });
 
+  it('should delete the article', () => {
```

Comment:
according to the task description you need to create separate command, to create article

Reply:
> according to the task description you need to create separate command, to create article

thanks, already fixed